inputPage.php

```diff
@@ -0,0 +1,62 @@
+<?php
+require_once('functions.php');
+$db = linkDB();
+$mice = getDataFromDB($db);
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <?php
+    require_once('HTMLhead.php');
+    ?>
+</head>
+
+<body>
+<nav>
+    <?php
+    require_once('HTMLnav.php');
+    ?>
+</nav>
+<main class="inputPageMain">
+    <h2>Add to the Mouse collection!</h2>
+    <div class = "inputPageDiv">
+        <form class="inputPageForm">
+            <label>
+                Mouse name:
+                <input name="mouseName" type="text" placeholder = "Enter Mouse Name Here" required/>
+            </label>
+            <label>
+                Brand:
+                <input name="mouseBrand" type="text" placeholder = "Enter Mouse Brand Here" required/>
```

Comment:
small stylistic note: at the moment, the input fields increase in size to fit the container when changing the browser window - not sure if this was intended or not but I thought I'd mention it anyway :)

Reply:
yeah haha i don't mind. hopefully not gonna get judged too badly on my poor taste.....